src/registry_schemas/example_data/schema_data.py

```diff
@@ -2568,6 +2568,59 @@
     }
 }
 
+CORRECTION_CP_SPECIAL_RESOLUTION = {
+    'filing': {
+        'header': {
+            'name': 'correction',
+            'certifiedBy': 'full name',
+            'date': '2023-06-07',
+            'folioNumber': '',
+            'waiveFees': True,
+            'priority': False
+        },
+        'business': {
+            'foundingDate': '2007-04-08T00:00:00+00:00',
+            'identifier': 'CP1234567',
+            'legalName': 'legal name - CP1234567',
+            'legalType': 'CP'
+        },
+        'correction': {
+            'legalType': 'CP',
+            'courtOrder': COURT_ORDER,
+            'correctedFilingId': 1234,
+            'correctedFilingType': 'specialResolution',
+            'cooperativeAssociationType': 'CP',
+            'rulesInResolution': True,
+            'rulesFileKey': 'cooperative/fa00c6bf-eaad-4a07-a3d2-4786ecd6b83b.jpg',
+            'memorandumFileKey': 'cooperative/f722bf16-86be-430d-928d-5529853a3a2c.pdf',
+            'rulesFileName': 'coop rules file name',
+            'rulesUploadedOn': '',
+            'memorandumInResolution': True,
```

Comment:
Maybe a comment explaining InResolution vs FileKey / FileName?

Technically this payload would fail because it's either one or the other?

Reply:
yes you're correct, I removed FileName, vysakh suggest remove it as well